packages/terra-application/src/application-navigation/private/common/_PopupMenuUser.jsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Avatar, { Generic } from 'terra-avatar';
 import ThemeContext from 'terra-theme-context';
+import Button from 'terra-button';
 
-import { userConfigPropType } from '../utils/propTypes';
+import { userConfigPropType, userActionConfigPropType } from '../utils/propTypes';
 
 import styles from './PopupMenuUser.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   *  An id for the user action utility button
```

Comment:
```suggestion
   *  An id for the user action button
```
`utility` might make this id to more associated to utility items then userAction button.